python/src/equistore/labels.py

```diff
@@ -251,6 +251,41 @@ def asarray(self):
     def __contains__(self, label):
         return self.position(label) is not None
 
+    @staticmethod
+    def arange(*args, name="name"):
```

Comment:
And explicit parameters please. Who knows what you want as `*args`.

Reply:
I would force an explicit name instead of having a default:

```
    @staticmethod
    def arange(name, *args):
```

or with explicit arguments

```
    @staticmethod
    def arange(name, start, stop=None, step=None):
```

Reply:
I've made the name mandatory and I've documented the three integer parameters